python/requirements/ml/requirements_dl.txt

```diff
@@ -10,11 +10,11 @@ keras==2.6.0
 tensorflow==2.6.0
 tensorflow-probability==0.14.0
 
-torch==1.9.0;sys_platform=="darwin"
-torchvision==0.10.0;sys_platform=="darwin"
+torch==1.11.0;sys_platform=="darwin"
```

Comment:
If you make changes here can you also make the corresponding changes to `requirements_ml_docker.txt`? Thanks!